src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ value }) => (
+  // <button className="button" type="submit">
```

Comment:
You need to remove the commented code, It’s better to delete the commented code. If you need to review a previous version of your code, use git history.